Sources/Stevenson/SlackService.swift

```diff
@@ -29,18 +29,21 @@ public struct SlackCommand {
 public struct SlackCommandMetadata: Content {
     public let token: String
     public let channelName: String
+    public let command: String
```

Comment:
Not sure I understand the need for that new property, can you elaborate?

Reply:
it is provided in the incoming request and having it as a property will allow us to have one command that would serve as a faced for more specific commands. i.e. the request will come with command `/stevenson` and text `fastlane ...`, we then can transform it to `fastlane` command, like we do with `hockeyapp` command now, when we drop `/fastlane` from it. If we don't go this route there is no harm in having this property.

Reply:
Ok I see, but I mainly saw it used with value `""` in the PR for now, which is why I found it confusing though.